docs/docs/parachains/composable/composable-council.md

```diff
@@ -0,0 +1,125 @@
+# A Look into the Composable Parachain's Governance
+
+Similar to the [governance of Picasso](../parachains/picasso/governance), Composable’s Polkadot parachain will ultimately become completely decentralized and democratic. This requires a sufficient user base, so in the initial stages of parachain deployment, the Composable Council will aid in governance. 
+
+:::info
+Governance processes are carried out via Substrate governance-affiliated democracy pallets, a component of the Kusama and Polkadot network runtimes that can be found within the Substrate libraries. Governance also involves referenda, which will ensure that the majority of our community supports any implemented changes and decisions. This further ensures transparency in alterations to the Composable parachain.
```

Comment:
Just add reference, do not name `pallet`, nobody care. Just `same democractic process` as on Polkdato and Kusama.

Reply:
edited